### PR TITLE
importenv enhancements

### PR DESCRIPTION
**Pull request by HarithaIBM**
This is fix for enhancement described in "https://github.com/ZOSOpenTools/meta/issues/333"

### CODE REVIEW

File: bin/zopen-importenvs [hunk 1]
```diff
@@ -37,19 +38,18 @@ then
 fi
   echo "final depsPath = $depsPath"
```
Comment (IgorTodorovskiIBM) [inline bin/zopen-importenvs:39]: This seems like debug output, should it be removed?

File: bin/zopen-importenvs [hunk 2]
```diff
@@ -37,19 +38,18 @@ then
...
-# then , the ports are fetched from $HOME/zopen/prod and $HOME/zopen/boot directory
-# then its env files are fetched and sourced, if they are not already sourced.
-if [ $# -eq 0 ]
-then
-     echo "Sourcing projects from $HOME/zopen/prod and $HOME/zopen/boot"
-     
-     # arrOfDirs, is an array which holds the names of dirs whose env is set
+
+sourceEnvFromPath () {
+    echo "sourceEnvFromPath enter, called with arg - $1"
```
Comment (IgorTodorovskiIBM) [inline bin/zopen-importenvs:43]: also looks like debug output, can it be removed?

**Review by IgorTodorovskiIBM — APPROVED** [review]
Looks good other than 2 minor comments

File: bin/zopen-importenvs (hunk 1 above)
Comment (HarithaIBM) [reply bin/zopen-importenvs]: yes, I have removed it.

File: bin/zopen-importenvs (hunk 2 above)
Comment (HarithaIBM) [reply bin/zopen-importenvs]: yes, I have removed it. Thank you.

**Review by IgorTodorovskiIBM — APPROVED** [review]
LGTM

**Pull request merged**